tests/test_hub_features.py

```diff
@@ -35,3 +38,34 @@ def test_subfolder(self):
             model = PeftModel.from_pretrained(model, model_id, subfolder=subfolder)
 
             assert isinstance(model, PeftModel)
+
+
+class TestBaseModelRevision:
+    def test_save_and_load_base_model_revision(self, tmp_path):
+        r"""
+        Test if subfolder argument works as expected
+        """
+        lora_config = LoraConfig(r=8, lora_alpha=16, lora_dropout=0.0, init_lora_weights=False)
+        test_inputs = torch.arange(10).reshape(-1, 1)
+
+        for model_id, revision in BASE_REVISION_MODELS_TO_TEST:
+            original_base_model = AutoModelForCausalLM.from_pretrained(model_id, revision="main").eval()
+            original_peft_model = get_peft_model(original_base_model, lora_config)
+            original_peft_sum = original_peft_model(test_inputs).logits.sum()
```

Comment:
Not really necessary to take the sum here, we can compare the logits directly.

Reply:
done